### PR TITLE
Make sure the text is similar when matching strings with same location but different IDs

### PR DESCRIPTION
**Pull request by strategio**
https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlpb-159

### CODE REVIEW

**Comment by strategio** [comment]
@brucepearson, you should review this.

File: src/st/class-wpml-pb-reuse-translations.php [hunk 1]
```diff
@@ -138,6 +138,16 @@ private function is_same_location_and_different_ids( array $current_string, arra
...
 	}
 
+	/**
+	 * @param string $old_text
+	 * @param string $new_text
+	 *
+	 * @return bool
+	 */
+	private function is_similar_text( $old_text, $new_text ) {
+		return 50 < WPML_ST_Diff::get_sameness_percent( $old_text, $new_text );
```
Comment (brucepearson) [inline src/st/class-wpml-pb-reuse-translations.php]: @strategio Please don't use yodo here. It makes it hard to read and is not needed. You only need yodo conditions when comparing with variables.

Comment (strategio) [reply src/st/class-wpml-pb-reuse-translations.php]: Done @brucepearson... you mean yoda :)

**Comment by strategio** [comment]
@OnTheGoSystems/wpml please review.

**Comment by strategio** [comment]
@OnTheGoSystems/wpml, one more review please :)

**Pull request merged**